### PR TITLE
py-spacy: new version 2.3.2

### PR DESCRIPTION
**Pull request by aweits**
update en-core-web-sm to @2.3.1
add en-vectors-web-lg@2.3.0

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-spacy/package.py [hunk 1]
```diff
@@ -9,8 +9,9 @@ class PySpacy(PythonPackage):
     Python and Cython."""
 
     homepage = "https://spacy.io/"
-    url      = "https://pypi.io/packages/source/s/spacy/spacy-2.2.4.tar.gz"
+    url      = "https://pypi.io/packages/source/s/spacy/spacy-2.3.2.tar.gz"
 
+    version('2.3.2', sha256='818de26e0e383f64ccbe3db185574920de05923d8deac8bbb12113b9e33cee1f')
     version('2.2.4', sha256='f0f3a67c5841e6e35d62c98f40ebb3d132587d3aba4f4dccac5056c4e90ff5b9')
 
     depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-spacy/package.py]: This latest version has some new dependency requirements:
```
python_requires = >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*
setup_requires =
        wheel
        cython>=0.25
        cymem>=2.0.2,<2.1.0
        preshed>=3.0.2,<3.1.0
        murmurhash>=0.28.0,<1.1.0
        thinc==7.4.1
install_requires =
        murmurhash>=0.28.0,<1.1.0
        cymem>=2.0.2,<2.1.0
        preshed>=3.0.2,<3.1.0
        thinc==7.4.1
        blis>=0.4.0,<0.5.0
        wasabi>=0.4.0,<1.1.0
        srsly>=1.0.2,<1.1.0
        catalogue>=0.0.7,<1.1.0
        tqdm>=4.38.0,<5.0.0
        setuptools
        numpy>=1.15.0
        plac>=0.9.6,<1.2.0
        requests>=2.13.0,<3.0.0
        pathlib==1.0.1; python_version < "3.4"
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-spacy/package.py [hunk 2]
```diff
@@ -9,17 +9,20 @@ class PySpacy(PythonPackage):
...
+    depends_on('python@2.7:2.8,3.5:', type=('build', 'run'), when='@2.3.0:')
     depends_on('py-wheel', type='build')
     depends_on('py-cython@0.25:', type='build')
     depends_on('py-murmurhash@0.28:1.0', type=('build', 'run'))
     depends_on('py-cymem@2.0.2:2.0.999', type=('build', 'run'))
     depends_on('py-preshed@3.0.2:3.0.999', type=('build', 'run'))
-    depends_on('py-thinc@7.4.0', type=('build', 'run'))
+    depends_on('py-thinc@7.4.0', type=('build', 'run'), when='@2.2.4:2.2.999')
+    depends_on('py-thinc@7.4.1', type=('build', 'run'), when='@2.3.0:')
     depends_on('py-blis@0.4.0:0.4.999', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-spacy/package.py:26]: Missing new dependency:
```
        wasabi>=0.4.0,<1.1.0
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-spacy/package.py (hunk 2 above)
Comment (aweits) [reply var/spack/repos/builtin/packages/py-spacy/package.py:26]: ah, that was a missing existing dep!

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/py-spacy/package.py:26]: Oops, my bad. I think I wrote this package, so I'm surprised I didn't notice it the first time.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**